src/utils/getDefaultCommitContent.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import fs from 'fs'
+
+const COMMIT_FILE_PATH_INDEX = 3
+const COMMIT_TITLE_LINE_INDEX = 0
+const COMMIT_MESSAGE_LINE_INDEX = 2
+
+const getDefaultCommitContent = () => {
```

Comment:
It would be cool to type the return value of this function

```suggestion
const getDefaultCommitContent = (): { title: type, message: type } => {
```

Reply:
Done